pkg/operator/status_test.go

```diff
@@ -0,0 +1,61 @@
+package operator
+
+import (
+	configv1 "github.com/openshift/api/config/v1"
+	osconfigv1 "github.com/openshift/api/config/v1"
+	fakeconfigclientset "github.com/openshift/client-go/config/clientset/versioned/fake"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func TestCheckMachineAPI(t *testing.T) {
+	tConditions := []struct {
+		expectedErr  error
+		expectedBool bool
+		conditions   []osconfigv1.ClusterOperatorStatusCondition
+	}{
+		{
+			expectedErr:  nil,
+			expectedBool: true,
+			conditions: []osconfigv1.ClusterOperatorStatusCondition{
+				{
+					Type:   configv1.OperatorAvailable,
+					Status: osconfigv1.ConditionTrue,
+				},
+				{
+					Type:   configv1.OperatorFailing,
+					Status: osconfigv1.ConditionFalse,
+				},
+			},
+		},
+		{
```

Comment:
This item is identical to the previous one. Is that intentional?

Reply:
Good catch!  No, I don't know how that happend!

Reply:
@ingvagabund fixed.